flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -1293,6 +1320,125 @@ private TableResult buildShowResult(String columnName, String[] objects) {
                 Arrays.stream(objects).map((c) -> new String[] {c}).toArray(String[][]::new));
     }
 
+    private String[] buildShowCreateTableRow(
+            ResolvedCatalogBaseTable<?> table,
+            ObjectIdentifier sqlIdentifier,
+            boolean isTemporary) {
```

Comment:
hive dialect has its own grammar to create table. Do we need to build table with hive dialect if the table is `Permannet`?